src/components/Alert/Alert.scss

```diff
@@ -0,0 +1,40 @@
+@use "../../assets/theme/macbis-colors" as c;
+
+.usa-alert {
+  button {
+    border: none;
+    background-color: transparent;
+  }
+}
+
+.usa-alert--info:before {
+  -webkit-mask: url("../../assets/img/usa-icons/info_alt.svg") no-repeat
+    center/contain;
+  mask: url("../../assets/img/usa-icons/info_alt.svg") no-repeat center/contain;
+}
+
+.usa-alert--success:before {
+  -webkit-mask: url("../../assets/img/usa-icons/success.svg") no-repeat
+    center/contain;
+  mask: url("../../assets/img/usa-icons/success.svg") no-repeat center/contain;
+}
```

Comment:
Check these in the build to make sure that they actually get pulled in correctly. I never got around to playing around with/fixing images in Sass files. Can't remember if this will work or not.

Reply:
Good call